src/schema.graphql

```diff
@@ -253,6 +261,13 @@ enum AccountStatus {
 }
 
 type Mutation {
+  requestPhoneCodeGeetest(
```

Comment:
Are we keeping this approach of a separate mutation, or should the captcha parameters be added to the original requestPhoneCode?